File: src/Entity/Devoir.php
<?php

namespace App\Entity;

use App\Repository\DevoirRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=DevoirRepository::class)
 */
class Devoir
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=50)
     */
    private $nomDevoir;

    /**
     * @ORM\Column(type="string", length=50)
     */
    private $description;


    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomDevoir()
    {
        return $this->nomDevoir;
    }

    public function setNomDevoir($nomDevoir)
    {
        $this->nomDevoir = $nomDevoir;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }


}

File: src/com/mycompany/myapp/entities/Devoir.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;



/**
 *
 * @author 21655
 */
public class Devoir {
    private int id;
    private String nomDevoir;
    private String description;

    public Devoir(int id, String nomDevoir, String description) {
        this.id = id;
        this.nomDevoir = nomDevoir;
        this.description = description;
    }

    public Devoir() {
    }

   
   

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNomDevoir() {
        return nomDevoir;
    }

    public void setNomDevoir(String nomDevoir) {
        this.nomDevoir = nomDevoir;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "Devoir{" + "id=" + id + ", nomDevoir=" + nomDevoir + ", description=" + description + '}';
    }
    


}
